api/go.mod

```diff
@@ -1,6 +1,7 @@
 module github.com/containerd/containerd/api
 
 go 1.21
+toolchain go1.24.1
 
```

Comment:
Why does the bot want to add this line?
🤔

Reply:
I think it's some weird behavior of dependabot.

Whenever the go min version needs a bump, instead of bumping go to the new minimum version, it add a `toolchain` with the latest go version.

Reply:
created a manual PR https://github.com/containerd/containerd/pull/11574